backend/examples/generate_signatures.rs

```diff
@@ -0,0 +1,34 @@
+#![feature(generic_const_exprs)]
+use std::{error::Error, fs::File};
+
+use csv::WriterBuilder;
+
+mod mock_signer;
+use mock_signer::sign_message;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    // You can modify the message to gain better trust from users, or simply follow CEX requirements.
+    // The message will be used to verify addresses and register them in the `ownershipProofByAddress` mapping on the Summa contract.
+    let message = "Summa proof of solvency for CryptoExchange";
+    let path = "src/apis/csv/signatures.csv";
+
+    // Generate signatures for the given 'message' using the mock signer.
+    // For this example, the 'mock_signer' file contains only the 'sign_message' function.
+    // CEX should implement their own signer and use it here instead of 'sign_message'.
+    let signatures = sign_message(message).await?;
+
+    // Write the signatures to a CSV file to be used in the `verify_signatures` example.
+    // It's envisioned that this CSV file will remain internal to CEX; only the Summa contract will publish its contents.
+    let file = File::create(path)?;
+    let mut wtr = WriterBuilder::new().delimiter(b';').from_writer(file);
```

Comment:
Why isn't a comma sufficient?

Reply:
Yeah, It's sufficient. I will fix this too.

Reply:
Fixed at [eacfe7d](https://github.com/summa-dev/summa-solvency/pull/159/commits/eacfe7df0542f35ab9367cc34ad8a298b0cdda5d)